docker/build-docker-images.sh

```diff
@@ -150,6 +147,6 @@ elif [[ "$TRAVIS_BRANCH" == "master" ]] || [[ "$TRAVIS_BRANCH" == *"-stable" ]];
   docker_push 'kuzzle' "$major_version-alpine"
   docker_push 'kuzzle' "$major_version-scratch"
 else
-  echo "Incorrect value for TRAVIS_BRANCH variable ("$TRAVIS_BRANCH"). Exiting."
+  echo "Incorrect value for BRANCH variable ("$BRANCH"). Exiting."
```

Comment:
The script should exit with a non-zero code 
```suggestion
  echo "Incorrect value for BRANCH variable ("$BRANCH"). Exiting."
  exit 1
```